added Tamil translation

tamil752002 opened the pull request:
I Just Added Tamil Language Translation,  If  You Like This , you Can Do merge

imblowfish commented on src/utils/translation.ts:
```diff
@@ -16,6 +16,7 @@ import gu from "./translations/gu";
 import vi from "./translations/vi";
 
 import { storage } from "../data/Storage";
+import ta from "./translations/ta";
```
Please import your localization file above like we doing this with other localization files

tamil752002 replied:
Do you mean to import the files in the correct order, similar to how the others are organized?

imblowfish commented on src/utils/translation.ts:
```diff
@@ -41,6 +43,9 @@ export async function init() {
     .use(intervalPlural)
     .init({
       resources: {
+        ta:{
```
Please place `ta` in the same sequence like in the imports

tamil752002 replied:
ok I Will Change That

tamil752002 replied:
I have done that

imblowfish commented on src/utils/translation.ts:
```diff
@@ -91,6 +96,7 @@ export async function init() {
         id: ["id"],
         gu: ["gu"],
         vi: ["vi"],
+        ta:["ta"],
```
Please use `npm run fmt` to fix formatting problems

tamil752002 replied:
I fixed This

imblowfish commented on src/utils/translations/ta.ts:
```diff
@@ -0,0 +1,102 @@
+    Period: "மாதவிடாய்",
+    today: "இன்று",
+    Delay: "தாமதம்",
+    Days: "(0)[நாள்];(1)[நாள்];(2-4)[நாள்];(5-20)[நாள்];(21)[நாள்];(22-24)[நாள்];(25-30)[நாள்];(31)[நாள்];(32-34)[நாள்];(35-40)[நாள்];",
```
You can use instead:

```ts
    Days: "(0-inf)[நாள்]", 
```

tamil752002 replied:
Okk , I Will Change That Properly

tamil752002 replied:
what is That mean "inf",
  Days: "(0)[நாள்];(1)[நாள்];(2-4)[நாள்];(5-20)[நாள்];(21)[நாள்];(22-24)[நாள்];(25-30)[நாள்];(31)[நாள்];(32-34)[நாள்];(35-40)[நாள்];",  Is This Wrong Format ?

imblowfish replied:
This is right format, but you don't need to write all thisniptions are neccessary in case of language where specific word depends on current number

As example, in Russian we say `21 день` or `32 дня`.
In English there will be `1 day`, `2 days` etc.

In your language you don't have any difference and use the same word for every number, so it's unnecesaary to write the same for every option number and instead you can use (0-inf) what means from zero to infinity

tamil752002 replied:
got it , I changed that , it works well, thank you guiding me :)

IraSoro reviewed (changes requested):
Hello! Can you please do what is described in section 8, 9 of the [CONTRIBUTING.md
](https://github.com/IraSoro/peri/blob/master/CONTRIBUTING.md) file

tamil752002 commented:
Hi maintainers,

I have made the changes based on the feedback provided 
- Addressed all the comments and fixed the issues.
- Updated the code to align with the project's guidelines.

Since I'm still new to contributing to open source, please forgive any mistakes I may have made. I am more than willing to address further feedback if needed. Thank you for your patience and guidance.

Looking forward to your review!

Best regards,
Tamilselvan R

IraSoro commented:
> Hi maintainers,
> 
> I have made the changes based on the feedback provided
> 
> * Addressed all the comments and fixed the issues.
> * Updated the code to align with the project's guidelines.
> 
> Since I'm still new to contributing to open source, please forgive any mistakes I may have made. I am more than willing to address further feedback if needed. Thank you for your patience and guidance.
> 
> Looking forward to your review!
> 
> Best regards, Tamilselvan R

@tamil752002, everythink is fine! Thank you for fixing the errors we wrote in the comments!

IraSoro commented on src/App.css:
```diff
@@ -44,7 +44,8 @@ ion-button {
   --border-radius: 16px;
   --box-shadow: 0 2px 6px 0 rgb(0, 0, 0, 0.25);
-  width: 150px;
+  min-width: 150px;
```
I guess you changed the width of the button because the word "mark" in Tamil is long. Is there another correct and short version to translate "Mark period"? If the word "அடையாளப்படுத்தவும்" is the most correct, then we'll leave it.

tamil752002 replied:
Which is why I used that word, but now I have changed it from 'அடையாளப்படுத்தவும்' to 'குறி, also I remove Css which i added Before

tamil752002 commented on src/utils/translations/ta.ts:
```diff
@@ -0,0 +1,103 @@
+  edit: "திருத்தவும்",
+  save: "சேமிக்கவும்",
+  "Period is": "மாதவிடாய்",
+  "possible today": "வாய்ப்பு இன்று உள்ளது",
```
What Is That Mean Encoding problem, could u expain More ,I am Wondering What Mistake I Made ,  It Would Be Helpful For me If You Provide That

imblowfish replied:
I'm really sorry. Looks like my laptop does't support this font well. I checked on my phone and on another laptop and everything looks fine. You don't need to change anything here
![image](https://github.com/user-attachments/assets/feae7922-384b-41e8-a1b6-15407ba81b7c)

tamil752002 replied:
Okk Thank You :)

IraSoro commented:
@tamil752002, thank you for adding Tamil language! And congratulations on your first contribution to the `Peri` project!

tamil752002 commented:
I am very glad to contribute to the Peri project, @IraSoro and @imblowfish  Thank you for guiding me
